packages/create-platformatic/package.json

```diff
@@ -21,7 +21,6 @@
   "license": "Apache-2.0",
   "author": "Matteo Collina <hello@matteocollina.com>",
   "dependencies": {
-    "@platformatic/authenticate": "workspace:*",
```

Comment:
please remove this module entirely

Reply:
Should I do it in this PR?

Reply:
Asking because this PR is not a bracking change yet.

Reply:
Anyway I creted another PR with a base to this one. It's up to you where to merge _this_ PR to the v1 or v2.
https://github.com/platformatic/platformatic/pull/2852